core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1066,6 +1066,23 @@ public void listStatus(AlluxioURI path, ListStatusContext context,
     return fileInfos;
   }
 
+  @Override
+  public void loadMetaData(AlluxioURI path, LoadMetadataContext context)
+      throws AccessControlException, FileDoesNotExistException, InvalidPathException, IOException {
+    try (RpcContext rpcContext = createRpcContext(context)) {
+      DescendantType syncDescendantType =
```

Comment:
can we invoke `loadMetadataIfNotExist` here? It looks to me some duplicated code with that method